core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -22,6 +22,7 @@
   String servicePathCartUpdateItems = "/V1/guest-carts/";
   String guestCartShippingInformationPath = "/us/V1/guest-carts/{cartId}/shipping-information";
   String guestCartCreateOrderPath = "/us/V1/guest-carts/{cartId}/order";
+  String pincodeLocationPath="/content/dam/hclecomm/en/json/state+district+city.json";
```

Comment:
@ReenaOmana : dont include opertors in json file name ; pl rename to state_district_city.json

Reply:
File has been renamed now